spinalcordtoolbox/vertebrae/detect_c2c3.py

```diff
@@ -74,39 +74,35 @@ def detect_c2c3(nii_im, nii_seg, contrast, nb_sag_avg=7.0, verbose=1):
     sct.run(cmd_detection, verbose=0, raise_exception=False)
 
     pred = nib.load('data_midSlice_pred_svm.hdr').get_data()
+    # save the prediction data before post-processing
+    nii_pred_before_postPro = nii_midSlice.copy()
+    nii_pred_before_postPro.data = pred
 
     # Create mask along centerline
     midSlice_mask = np.zeros(midSlice_seg.shape)
     mask_halfSize = int(np.rint(25.0 / nii_midSlice.dim[4]))
     for z in range(midSlice_mask.shape[1]):
```

Comment:
usually we operate in RPI orientation, meaning that S-I axis would correspond to .shape[2] (not .shape[1]). Could you add comments that would help understanding the code (e.g. what orientation is assumed here)